packages/product-viewer/src/product-editor.ts

```diff
@@ -26,10 +26,13 @@ import { LayoutMixin } from "./features/Layout";
 import { LightingMixin } from "./features/Lighting";
 import { LoaderMixin } from "./features/Loader";
 import { ScaleReferenceMixin } from "./features/ScaleReference";
+import { WireframeMixin } from "./features/Wireframe";
 import ProductViewerElementBase from "./product-viewer-base";
 
 export const ProductEditorElement = InspectorMixin(
-	ScaleReferenceMixin(LightingMixin(LoaderMixin(CameraMixin(LayoutMixin(ARMixin(ProductViewerElementBase)))))),
+	WireframeMixin(
+		ScaleReferenceMixin(LightingMixin(LoaderMixin(CameraMixin(LayoutMixin(ARMixin(ProductViewerElementBase)))))),
```

Comment:
Is there any other way to combine these mixins? This list is already a bit hard to read and would only get worse as we add more

Reply:
Yeah, putting them in a list and looping over it would at least appear a little cleaner, though not sure if it would affect the typing. I'll try it out

Reply:
If looping doesn't work, another pattern I've seen is:
```
export const ProductEditorElement = 
    InspectorMixin(
    WireframeMixin(
	ScaleReferenceMixin(
    LightingMixin(
    LoaderMixin(
    CameraMixin(
    LayoutMixin(
    ARMixin(
        ProductViewerElementBase
    )))))),
	)
);
```

It has it's own issues but might be better than a single line if we have a lot of mixins.